packages/app-store/routing-forms/pages/reporting/[...appPages].tsx

```diff
@@ -168,10 +167,10 @@ const Reporter = ({ form }: { form: inferSSRProps<typeof getServerSideProps>["fo
   return (
     <div className="cal-query-builder">
       <Query
-        {...config}
+        {...config as unknown as Config}
         value={query.state.tree}
         onChange={(immutableTree, config) => {
-          onChange(immutableTree, config as QueryBuilderUpdatedConfig);
+          onChange(immutableTree, config as unknown as FormFieldsQueryBuilderConfigWithRaqbFields);
```

Comment:
QueryBuilder gives a generic config but we know we gave it FormFieldsQueryConfig.

I will see later how I can remove this unknown type assertions.

Reply:
Maybe a transformer function that loosens the typing